metashare/api/views.py

```diff
@@ -47,6 +47,17 @@ def post(self, request):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class UserDisconnectSFView(CurrentUserObjectMixin, APIView):
+    model = User
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request):
+        user = self.get_object()
+        user.invalidate_salesforce_credentials()
+        serializer = FullUserSerializer(user)
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
@wlonk Note that I changed this to return the serialized User to avoid either an extra API call or making assumptions about the form of the updated data.